### PR TITLE
Update README.md to include more info about why we built it

### PR DESCRIPTION
**Pull request by ImMattDavison**
This PR updates the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
-<h1 align='center'>Life's a Beach</h1>
+<h1 align='center'>Life's a Beach 🏖️</h1>
 
 <p align='center'><i>Don't get washed away in the sea of terminology</i></p>
```
Comment (AdamDIOM) [inline README.md]: Uncapitalise Don't please

File: README.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
 
 ![Douglas Beach](https://user-images.githubusercontent.com/66436400/246520763-936f6ccb-135d-4ded-95a5-487f2c7c816e.jpeg)
 
+## Why we built Life's a Beach! 🌊
+
+PrideHacks should be all about celebrating and understanding LGBT+ identities. As someone who identifies as LGBT+ (Matt) I understand just how difficult some people can find it to understand LGBT+ identities, no matter how much they'd love to. I think it is incredibly important that good educational content is available to support people in learning about the people they love and those that matter around them.
+
+Why Life's a Beach? Well Adam wanted it to have beaches, so there's beaches. To add to this! Beaches are calm, relaxing places which I have found to be a serene place whenever I feel stressed, unwanted or unloved. I can sit on the sand or walk through the sea and feel at peace. ~ Adam
```
Comment (AdamDIOM) [inline README.md]: Add _ before "To add to this!" And after "~ Adam" please

File: README.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+
+## Additional info ℹ️
+
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
 Run the project locally using `npm run dev` and open [http://localhost:3000](http://localhost:3000) with your browser to utilise it's features.
```
Comment (AdamDIOM) [inline README.md]: No apostrophe for its

**Review by AdamDIOM — CHANGES_REQUESTED** [review]
See inline comments

File: README.md (hunk 1 above)
Comment (ImMattDavison) [reply README.md]: Can you please do it - I am no longer at my computer :)

**Review by AdamDIOM — APPROVED** [review]

**Pull request merged**